yarn-project/simulator/src/simulator/acvm_native.ts

```diff
@@ -40,8 +40,13 @@ function parseIntoWitnessMap(outputString: string) {
     lines
       .filter((line: string) => line.length)
       .map((line: string) => {
-        const pair = line.replaceAll(' ', '').split('=');
-        return [Number(pair[0]), pair[1].replaceAll('"', '')];
+        try {
+          const pair = line.replaceAll(' ', '').split('=');
+          return [Number(pair[0]), pair[1].replaceAll('"', '')];
+        } catch (error) {
+          logger.debug(line);
+          return [0, ''];
+        }
```

Comment:
There are instances where acvm returns a single value instead of a `a="b"` formatted string? Would it be worth using a TOML parser?

Reply:
No, this is left from some debugging. Have removed.